### ISSUE TITLE
Better documentation for JSON databags

### ISSUE THREAD

**Issue by Vesuvium** [lektor/lektor-website#150]
JSON databags have a weird behaviour and there should be an example of a simple JSON databag to show the correct structure of the file

**Comment by andrewsanchez** [comment]
Yes, I was just trying to figure this out myself and am getting weird results.  If you can elaborate here and provide some suggestions, I will try to submit a PR.

This is the page I'm trying to use my databag with:  http://andrewsanchez.me/cv/

https://github.com/andrewsanchez/andrewsanchez.github.io
